Client/TeamTalkClassic/gui/FilesTab.cpp

```diff
@@ -222,6 +222,10 @@ BOOL CFilesTab::PreTranslateMessage(MSG* pMsg)
 {
     if (WM_KEYFIRST <= pMsg->message && pMsg->message <= WM_KEYLAST)
         if (m_hAccel && ::TranslateAccelerator(AfxGetMainWnd()->m_hWnd, m_hAccel, pMsg))
+            CString szCtrlName;
+            szCtrlName.LoadString(IDS_FILESLISTLAB);

```

Comment:
This code should happen during initialisation, i.e. OnInitDialog(). Probably at line 183.